deploy/cstor-operator.yaml

```diff
@@ -3886,195 +3840,214 @@ spec:
+          image: openebs/node-disk-operator:1.7.0
+          imagePullPolicy: IfNotPresent
+          env:
+            - name: WATCH_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            # the service account of the ndm-operator pod
+            - name: SERVICE_ACCOUNT
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.serviceAccountName
+            - name: OPERATOR_NAME
+              value: "node-disk-operator"
+            - name: CLEANUP_JOB_IMAGE
+              value: "openebs/linux-utils:3.0.0"
+            # OPENEBS_IO_IMAGE_PULL_SECRETS environment variable is used to pass the image pull secrets
+            # to the cleanup pod launched by NDM operator
+            #- name: OPENEBS_IO_IMAGE_PULL_SECRETS
+            #  value: ""
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 8585
+            initialDelaySeconds: 15
+            periodSeconds: 20
+          readinessProbe:
+            httpGet:
+              path: /readyz
+              port: 8585
+            initialDelaySeconds: 5
+            periodSeconds: 10
 ---
 # Create NDM cluster exporter deployment.
 # This is an optional component and is not required for the basic
 # functioning of NDM
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: ndm-cluster-exporter
+  name: openebs-ndm-cluster-exporter
   namespace: openebs
+  labels:
+    name: openebs-ndm-cluster-exporter
+    openebs.io/component-name: ndm-cluster-exporter
+    openebs.io/version: dev
 spec:
   replicas: 1
   strategy:
     type: Recreate
   selector:
     matchLabels:
-      name: ndm-cluster-exporter
+      name: openebs-ndm-cluster-exporter
+      openebs.io/component-name: ndm-cluster-exporter
   template:
     metadata:
       labels:
-        name: ndm-cluster-exporter
+        name: openebs-ndm-cluster-exporter
+        openebs.io/component-name: ndm-cluster-exporter
+        openebs.io/version: dev
     spec:
-      serviceAccountName: openebs-maya-operator
+      serviceAccountName: openebs-cstor-operator
       containers:
-      - name: ndm-cluster-exporter
-        image: openebs/node-disk-exporter:ci
-        command:
-        - /usr/local/bin/exporter
-        args:
-        - "start"
-        - "--mode=cluster"
-        - "--port=$(METRICS_LISTEN_PORT)"
-        - "--metrics=/metrics"
-        ports:
-        - containerPort: 9100
-          protocol: TCP
-          name: metrics
-        imagePullPolicy: IfNotPresent
-        env:
-        - name: NAMESPACE
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.namespace
-        - name: METRICS_LISTEN_PORT
-          value: :9100
+        - name: ndm-cluster-exporter
+          image: openebs/node-disk-exporter:1.7.0
```

Comment:
By any chance.. is this YAML also used by the CI tests. In that case - we probably need this to be "ci".

Reply:
yes, we have to use `ci` tag

Reply:
Updated to `ci` tag